### ISSUE TITLE
fileLoader: incorrect behavior loading js files

### ISSUE THREAD

**Issue by maidanskyi** [Urigo/merge-graphql-schemas#269]
Hi, guys!

I'm using this package within my TypeScript project.
**Problem:** `fileLoader` includes `*.js` and `*.js.map` files in bundle after compiling TS -> JS
**Expected behavior:** Do not include `.js.map` files.

So, I've used
```
{
...
"sourceMap": true, // of course, it works correctly without this one)
...
}
```
in my `tsconfig.json`.
And here is my config for `fileLoader`:
```
const typeDefsArray = this
  .mergeTools
  .fileLoader(
    path.join(__dirname, '../../../components/**/typeDef.*'), {
      extensions: ['.ts', '.js'],
    },
  )

const resolversArray = this
  .mergeTools
  .fileLoader(
    path.join(__dirname, '../../../components/**/resolver.*'), {
      extensions: ['.ts', '.js'],
    },
)
```

**Comment by ardatan** [comment]
@maidanskyi Could you try with the following canary version?
`1.7.1-6dd08a4.0`

**Comment by maidanskyi** [comment]
@ardatan I've checked and there is the same problem

**Comment by ardatan** [comment]
@maidanskyi Could you try with this version one more time?
`1.7.1-7e201aa.0`

**Comment by maidanskyi** [comment]
@ardatan yeah, now it works!)
That'll be great to merge it asap

**Comment by ardatan** [comment]
Available in 1.7.2 !
